flask_rest_orm/resources/utils.py

```diff
@@ -22,10 +48,34 @@ def query_from_request(model, request):
     :return: the items for the current page
     """
     builder = model.query
-    for key in request.args:
-        if hasattr(model, key):
-            vals = request.args.getlist(key)  # one or many
-            builder = builder.filter(getattr(model, key).in_(vals))
+
+    def get_operator(attr, key, value):
+        if not key:
+            return attr.operate(operator.eq, value)
+        elif key in OPERATORS:
+            op = OPERATORS.get(key)
+            if op == 'between':
+                return attr.between(value[0],value[1])
+            return getattr(attr, op)(value)
+        elif key in OPERATE:
+            op = OPERATE.get(key)
+            return attr.operate(getattr(operator, op), value)
+        else:
+            raise Exception('Unknown operator')
+
+    def filter(attr, value):
```

Comment:
`build_query` would be a better name for the function. `request_filter` a better name for `value`

Reply:
OK